### ISSUE TITLE
Start a game on new map always moves Window to center of screen in SDL 2 mode

### ISSUE THREAD

**Issue by ihhub** [ihhub/fheroes2#264]
Open the game in SDL 2 mode. Move Window out of center and start a new game. You would notice that the window moves the center of screen which is weird.

**Comment by idshibanov** [comment]
@ihhub function that does it is `Game::SetFixVideoMode`. Do you know why we need it in the first place? It also forces me back into fullscreen if I pressed F4 in main menu.

**Comment by ihhub** [comment]
Hi @idshibanov , sorry for a very late reply. I missed your message. Honestly, I do not know the logic behind this function to be called multiple times during runtime. However, once we add resolution changing within the game this function would be handy.

I will make a fix for this particular bug.

**Comment by ihhub** [comment]
> @ihhub function that does it is `Game::SetFixVideoMode`. Do you know why we need it in the first place? It also forces me back into fullscreen if I pressed F4 in main menu.

Actually, `SetVideoMode` is responsible for creation of a new window and putting it into center of the screen.

**Comment by idshibanov** [comment]
Yes, I noticed that.. there's a lot of weird behavior currently regarding display modes. I also have a suspicion that scaled sprites (in main menu) are not handled properly in some cases and are related to fullscreen issues in #419.

For testing I've set display flag to SDL_WINDOW_RESIZABLE and game window handles it surprisingly well, but in main menu there's a ton of off behavior. Didn't get a chance to narrow it down.

@ihhub All of this might need a new issue open.
